nilearn/datasets/tests/test_atlas.py

```diff
@@ -691,6 +697,11 @@ def test_fetch_atlas_surf_destrieux(tmp_path):
     assert bunch.description != ""
 
 
+@pytest.mark.xfail(reason="improper mock data shape")
```

Comment:
To fix?

Reply:
+1

Reply:
Will try to "fix" but could mean having to start mocking some of the datasets we use.

Reply:
yep all fetchers are mocked in tests and you can control what the mock returns

Reply:
yeah but for one of those I was getting an "empty file" error: will try to investigate a bit more